packages/vaadin-themable-mixin/test/themable-mixin.test.js

```diff
@@ -11,15 +11,14 @@ const createStyles =
 
 const defineCustomElement =
   window.defineCustomElementFunction ||
-  ((name, parentName, content, styles) => {
+  // eslint-disable-next-line max-params
+  ((name, parentName, content, styles, noIs) => {
```

Comment:
Changed the `defineCustomElementFunction` test helpers to not include `is` and `template` in the class by default. Instead, they're added conditionally based on the arguments. A similar change was done to the test helper in "lit-setup.js"